src/plugins/data_source_management/common/index.ts

```diff
@@ -7,3 +7,6 @@ export const PLUGIN_ID = 'dataSourceManagement';
 export const PLUGIN_NAME = 'Data sources';
 export const DEFAULT_DATA_SOURCE_UI_SETTINGS_ID = 'defaultDataSource';
 export * from './types';
+export const DATA_SOURCE_SAVED_OBJECT_TYPE = 'data-source';
+export const DATA_SOURCE_PERMISSION_CLIENT_WRAPPER_ID = 'data-source-permission';
+export const ORDER_FOR_DATA_SOURCE_PERMISSION_WRAPPER = 50;
```

Comment:
Better to add some comment here, I guess you give it 50 to make it run behind all of other wrappers.

Reply:
Sure, I will add a comment